package.json

```diff
@@ -41,6 +41,7 @@
     "@koa/cors": "3.1.0",
     "@koa/multer": "3.0.0",
     "@koa/router": "10.0.0",
+    "@types/jest": "^26.0.24",
```

Comment:
Why did you add this package ? It does not seem to be used

Reply:
it's for auto-completion on Vs-code